slides/04-networking-in-riot/networking-in-riot.md

```diff
@@ -333,12 +347,12 @@ ssize_t sendto(int sockfd, const void *buf, size_t len, int flags,
 
 ## Exercise: Posix UDP client-server application
 
-Follow the instructions in the notebook **riot/networking/udp/udp.ipynb**
+Follow the instructions in the notebook **riot/ble/ble_scan/ble_scan.ipynb**
```

Comment:
Not sure I understand why this is changed to a ble notebook in the POSIX UDP client-server application.

Reply:
its a mistake on my side..